R/axis_geo.R

```diff
@@ -404,77 +407,86 @@ axis_geo <- function(
     # get the requested data if necessary
     scale_intervals <- intervals[[scale]]
     if (!is(scale_intervals, "data.frame")) {
-      scale_intervals <- get_scale_data(scale_intervals)
+      # remove trailing s for backwards compatibility
+      if (sub("s+$", "", scale_intervals) %in% c("period", "epoch", "era",
+                                                 "stage", "eon")) {
+        scale_intervals <-
+          get_timescale_data(name = "GTS2020",
+                             rank = sub("s+$", "", scale_intervals))
+      } else {
+        scale_intervals <- get_timescale_data(name = scale_intervals)
+      }
     }
-    # fix column names if using palaeoverse data
-    colnames(scale_intervals)[colnames(scale_intervals) == "interval_name"] <-
-      "name"
-    colnames(scale_intervals)[colnames(scale_intervals) == "max_ma"] <-
-      "max_age"
-    colnames(scale_intervals)[colnames(scale_intervals) == "min_ma"] <-
-      "min_age"
-    colnames(scale_intervals)[colnames(scale_intervals) == "colour"] <-
-      "color"
-    colnames(scale_intervals)[colnames(scale_intervals) == "lab_colour"] <-
-      "lab_color"
+    # fix column names if using deeptime data
```

Comment:
Is this required here? Isn't this handled in `get_timescale_data`?

Reply:
This handles deeptime-formatted timescales

Reply:
I guess my point with this comment was whether we would still expect deeptime-formatted timescales as input intervals? Presumably users will not get their data via deeptime to use axis_geo when they could use time_bins in palaeoverse? It's fine though if it allows a wider handling.